GoogleAppMeasurement.podspec.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This file should be removed.

Reply:
This is needed for travis until 5.9.0 goes public.

Reply:
SGTM, my bad!